airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -43,7 +43,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DefaultNormalizationRunner.class);
 
-  public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:0.1.0";
+  public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:dev";
```

Comment:
Shouldn't this be in some config files or based on env variables so we can switch from dev mode to prod easily or something?